content/rules.md

```diff
@@ -36,6 +36,7 @@ toy animals or party hats (provided they do not violate any of the above constra
 team needs at the contest, please contact the Head of Jury.
 
 Teams are allowed to bring one keyboard to replace the keyboard provided by the notebook. Note that keyboards with pre-programmed
-macro keys are not allowed. Wireless keyboards are also not allowed. A (single) mouse is allowed too, but is subject to the same macro rule as with the keyboard.
+macro keys are not allowed. Wireless keyboards are also not allowed.
+A (single) mouse is allowed too, but is subject to the rules as the keyboard (must be wired, must not be programmable).
```

Comment:
It is mentioned on the Systems page that we provide a mouse, so we may also want to repeat that here?

```suggestion
It is also allowed to bring a mouse to replace the mouse that we provide, but is subject to the rules as the keyboard (must be wired, must not be programmable).
```